bridge/core/import.go

```diff
@@ -0,0 +1,114 @@
+package core
+
+import "fmt"
+
+type ImportEvent int
+
+const (
+	_ ImportEvent = iota
+	ImportEventBug
+	ImportEventComment
+	ImportEventCommentEdition
+	ImportEventStatusChange
+	ImportEventTitleEdition
+	ImportEventLabelChange
+	ImportEventIdentity
+	ImportEventNothing
+)
+
+// ImportResult is an event that is emitted during the import process, to
+// allow calling code to report on what is happening, collect metrics or
+// display meaningful errors if something went wrong.
+type ImportResult struct {
+	Err    error
+	Event  ImportEvent
+	ID     string
+	Reason string
+}
+
+func (er ImportResult) String() string {
+	switch er.Event {
+	case ImportEventBug:
+		return "new issue"
+	case ImportEventComment:
+		return "new comment"
+	case ImportEventCommentEdition:
+		return "updated comment"
+	case ImportEventStatusChange:
+		return "changed status"
+	case ImportEventTitleEdition:
+		return "changed title"
+	case ImportEventLabelChange:
+		return "changed label"
+	case ImportEventIdentity:
+		return "new identity"
+	case ImportEventNothing:
+		return fmt.Sprintf("no event: %v", er.Reason)
+	default:
+		panic("unknown import result")
```

Comment:
> ExportResult actually has the same problem, String() will panic with the result of NewImportError() (that is, a perfectly legal value).

I saw this issue when i tried to use the import functionalities. Handling this case will be so usefull i think !

> `MergeResult` also 

I will make changes in the `entity` package also

Reply:
Should i add a new type `ImportEventError` and same for other event types ? 

Since the `NewError` implementation doesn't have any specific type

```golang
func NewImportError(err error, reason string) ImportResult {
	return ImportResult{
                // ImportType is 0 here
		Err:    err,
		Reason: reason,
	}
}
```

Reply:
Yeah, a new type of event make sense

Reply:
Done for all export, import and merge EventTypes